hudi-client/hudi-client-common/pom.xml

```diff
@@ -120,6 +120,16 @@
       <groupId>io.prometheus</groupId>
       <artifactId>simpleclient_pushgateway</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.uber.m3</groupId>
+      <artifactId>tally-m3</artifactId>
```

Comment:
Is there any risk for the uber specific jar to conflict with others?

Reply:
yeah, shading is fine.